hangrybot.py

```diff
@@ -88,6 +95,36 @@ def remind_to_go(self, channel="#lunch"):
             text=text
         )
 
+    def gorgonzola(self):
+        # get daily menus
+        bc = BackmarktCrawler()
+        menu_bc = bc.get_menus()
+        cc = CoronaCrawler()
+        menu_cc = cc.get_menus()
+
+        # search for gorgonzola
+        text = ''
+        if "gorgonzola" in menu_bc.lower() and "gorgonzola" in menu_cc.lower():
+            text = 'It has to be either Corona or Backmarkt!:star-struck:'
```

Comment:
I find it cool to use the unicode emojis in source, you can copy it in from:
https://emojipedia.org/emojipedia/5.0/grinning-face-with-star-eyes/